src/azure-cli-core/azure/cli/core/commands/constants.py

```diff
@@ -30,3 +30,5 @@
 DEFAULT_QUERY_TIME_RANGE = 3600000
 
 BLOCKED_MODS = ['context', 'shell', 'documentdb', 'component']
+
+CREDENTIAL_WARNING_EXCLUSIVE_COMMANDS = ['ad sp create-for-rbac']
```

Comment:
`az ad app credential reset` and `az ad sp credential reset` also print 

https://github.com/Azure/azure-cli/blob/38eaebb6936a32be13e2c4afba8e600ce9c91513/src/azure-cli/azure/cli/command_modules/role/custom.py#L49-L51

https://github.com/Azure/azure-cli/blob/38eaebb6936a32be13e2c4afba8e600ce9c91513/src/azure-cli/azure/cli/command_modules/role/custom.py#L1766

Should we remove duplicated warnings for them as well?

Reply:
Thanks for pointing out. I'll add them to the exclusive list as well